components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -31,7 +31,7 @@ export default function RepositoryFinder(props: RepositoryFinderProps) {
                 setSuggestedContextURLs(urls);
                 saveSearchData(urls);
             });
-    }, [suggestedContextURLs]);
+    }, []);
```

Comment:
Since this effect updates `suggestedContextURLs`, having it as a dep caused it to constantly run as soon as it calls `setSuggestedContextURLs`